### DISCUSSION TITLE
Great work! 👍

### DISCUSSION THREAD

**Discussion by XSAM** [connectrpc/connect-go#488] [Show and tell]
I just found this repository. And this is something I have dreamed of for years. Every change in the gRPC protocol that formed the [Connect protocol](https://connect.build/docs/protocol) is precisely what I want!

I'll use Connect protocol in my following projects. Thank you for making this happen. Great work, guys! 🙌

**Comment by akshayjshah** [comment]
Thanks for the kind words! 😍
